### PR TITLE
PL : Add even more channels

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Added most of the Polsat channels, not finished yet. 
Astra streams.

### CODE REVIEW

File: channels/pl.m3u [hunk 1]
```diff
@@ -1,16 +1,68 @@
...
+#EXTINF:-1 tvg-id="AXN.pl" tvg-name="AXN" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN (Poland) (1080p)
+http://91.218.60.3:8001/play/a00i
+#EXTINF:-1 tvg-id="AXN.pl" tvg-name="AXN" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN (Poland) (576p)
+http://91.218.60.3:8001/play/a00j
+#EXTINF:-1 tvg-id="AXNSpin.pl" tvg-name="AXN Spin" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN Spin (Poland) (1080p)
+http://91.218.60.3:8001/play/a00k
+#EXTINF:-1 tvg-id="AXNSpin.pl" tvg-name="AXN Spin" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN Spin (Poland) (576p)
+http://91.218.60.3:8001/play/a00l
+#EXTINF:-1 tvg-id="AXNWhite.pl" tvg-name="AXN White" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN White (Poland) (576p)
+http://91.218.60.3:8001/play/a00m
```
Comment (freearhey) [inline channels/pl.m3u:19]: AXN is US channel: https://www.lyngsat.com/tvchannels/us/AXN-Polska.html

File: channels/pl.m3u [hunk 2]
```diff
@@ -1,16 +1,68 @@
...
+#EXTINF:-1 tvg-id="CIPolsat.pl" tvg-name="CI Polsat" tvg-country="PL" tvg-language="" tvg-logo="" group-title="",CI Polsat (1080p)
```
Comment (freearhey) [inline channels/pl.m3u:20]: This is also the US channel: https://www.lyngsat.com/tvchannels/us/Crime---Investigation-Polsat.html

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/pl.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/pl.m3u:19]: Well AXN has got different divisions. While it is owned by Sony, the Polish versions is operated by AXN Europe, hence the fact that I didn't really know where to put it.

File: channels/pl.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/pl.m3u:20]: Noted, will move it.

**Review by freearhey — APPROVED** [review]

**Pull request merged**